plan/refactor-news-pages.md

```
@@ -0,0 +1,1042 @@
+       "aspect-[16/9] w-full object-cover transition-transform group-hover:scale-105";
+     ```
+
+     with:
+
+     ```tsx
+     className =
+       "aspect-[16/9] w-full object-cover transition-transform group-hover:scale-[1.03]";
+     ```
+
+  3. **Title hover color** (~line 156). In the `<h3>` className, replace:
+     ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)